### PR TITLE
Update proposal

### PR DESCRIPTION
**Pull request by imobachgs**
Merge #64 into `master`. It depends on https://github.com/yast/yast-packager/pull/224.

### CODE REVIEW

File: src/clients/packages_proposal.rb [hunk 1]
```diff
@@ -145,26 +149,36 @@ def main
           )
         )
 
+        @ret = {
```
Comment (teclator) [inline src/clients/packages_proposal.rb]: we could also initialize "preformatted_proposal"

File: src/clients/packages_proposal.rb [hunk 2]
```diff
@@ -145,26 +149,36 @@ def main
...
         elsif Ops.greater_than(Builtins.size(@warning), 0)
           # the proposal for the packages requires manual intervention
-          @ret = {
+          @ret.merge!({
             "preformatted_proposal" => HTML.List(@tmp),
             "warning"               => Builtins.mergestring(@warning, "<br>"),
             "warning_level"         => :warning
-          }
+          })
         else
```
Comment (teclator) [inline src/clients/packages_proposal.rb]: In case of initialized, then this else is unnecessary.

File: src/clients/packages_proposal.rb (hunk 1 above)
Comment (imobachgs) [inline src/clients/packages_proposal.rb]: Good point!

**Comment by teclator** [comment]
LGTM, thnx!

**Pull request merged**